mftutor/reg/views.py

```diff
@@ -38,6 +39,11 @@ class BurStartView(TemplateView):
 
 # =============================================================================
 
+class KrydslisteView(TemplateView):
+    template_name = 'reg/krydsliste.html'
```

Comment:
Denne skal være en `FormView` i stedet for `TemplateView`, med `form_class` sat til at være en formular med et felt der hedder `value`. Så skal du implementere `form_valid()` metoden til at kalde din `PDF` klasse.

Reply:
> Denne skal være en `FormView` i stedet for `TemplateView`, med `form_class` sat til at være en formular med et felt der hedder `value`. Så skal du implementere `form_valid()` metoden til at kalde din `PDF` klasse.

Jeg har lavet en løsning der virker, men det føles lidt hacky, måske du har en bedre løsning?